docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -430,21 +440,27 @@ And one additional optional argument:
  
 Because `calibrate_model_multistart` handles initial guess sampling, unlike for `calibrate_model`, no initial guess has to be provided. 
 
-Here, we fit parameters through 10 independent optimisation runs, using QuasiMonteCarlo's `SobolSample` method, and save the result to the OptimizationRuns folder:
+Here, we fit parameters through 10 independent optimisation runs, using QuasiMonteCarlo's `SobolSample` method, and save the result to the OptimisationRuns folder:
 ```@example petab1
 using Optim
 using QuasiMonteCarlo
-res_ms = calibrate_model_multistart(petab_problem, IPNewton(), 10, "OptimizationRuns"; sampling_method=QuasiMonteCarlo.SobolSample())
+mkdir("OptimisationRuns") # hide
```

Comment:
I think this needs a better path, otherwise won't it make this folder within the folder where this markdown file is located? We wouldn't want that I'd think.